czishrink/netczicompress/AppComposer.cs

```diff
@@ -72,7 +73,8 @@ public static MainViewModel ComposeMainViewModel(IProgramNameAndVersion info)
         var aggregateStatisticsViewModel = new AggregateStatisticsViewModel(
             aggregateIndicationViewModel: aggregateIndicationViewModel,
             ClipboardHelper.Instance,
```

Comment:
Not related to this PR actually, but wouldn't it be more consistent to use the named argument style also for the ClipboardHelper?

Reply:
Yes it would be. Sounds like a good excuse to create another PR 😄